source/includes/5.0-changes/fact-odd-number-releases.rst

```diff
@@ -0,0 +1,6 @@
+Prior to MongoDB 5.0, odd-numbered MongoDB release versions, such as
+``4.3``, were development releases. Beginning with MongoDB 5.1,
+MongoDB will have quarterly rapid releases. For more information on the
```

Comment:
Should we use present tense here? Otherwise this ages badly once we are past say 6.0